blackmarket/Common/Models/Session.swift

```diff
@@ -1,19 +1,32 @@
 import Foundation
 import RSSwiftNetworking
+import RSSwiftNetworkingAlamofire
 
-struct Session: Codable {
-  var uid: String?
-  var client: String?
-  var accessToken: String?
-  var expiry: Date?
+class Session: NSObject, Codable {
+  @objc dynamic var uid: String?
+  @objc dynamic var client: String?
+  @objc dynamic var accessToken: String?
+  @objc dynamic var expiry: Date?
```

Comment:
what is the motivation to use use NSObject and @objc dynamic here?

Reply:
I brought it without thinking about it from the base project, I don't need them so I took them out